apps/frontend/src/app/Components/FieldDisplay.tsx

```diff
@@ -121,25 +124,49 @@ export function NodeFieldDisplay({
   const { multi } = node.info
 
   const multiDisplay = multi && <span>{multi}&#215;</span>
+  const nodeValue = node.value
   let fieldVal = false as ReactNode
-  if (oldValue !== undefined) {
-    const diff = node.value - oldValue
-    const pctDiff = valueString(Math.abs(diff / oldValue), '%', node.info.fixed)
+  if (compareValue !== undefined) {
+    const diff = nodeValue - compareValue
+    const pctDiff = valueString(diff / compareValue, '%', node.info.fixed)
     fieldVal = (
       <>
-        <span>{valueString(oldValue, node.info.unit, node.info.fixed)}</span>
+        <span>{valueString(nodeValue, node.info.unit, node.info.fixed)}</span>
         {Math.abs(diff) > 0.0001 && (
-          <>
-            <ColorText color={diff > 0 ? 'success' : 'error'}>
-              {diff > 0 ? '+' : ''}
-              {valueString(diff, node.info.unit, node.info.fixed)}
+          <Tooltip
```

Comment:
This tooltip has bad contrast
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/fb5cb4f6-8515-421f-a793-4d8ac85725f7)
Can we use bootstraptooltip instead for the black background, and to match our other tooltips?